saleor/product/models/base.py

```diff
@@ -235,6 +241,7 @@ def get_first_category(self):
 
     def is_available(self):
         # type: () -> bool
+        """Checks if product is available today for customers"""
```

Comment:
That's why this function should not be here and should instead be merged into the availability info. Someone is bound to use it and shoot themselves in the foot.